internal/pkg/cli/task_run.go

```diff
@@ -108,6 +109,8 @@ type runTaskVars struct {
 
 	envVars                  map[string]string
 	secrets                  map[string]string
+	SSMParamSecrets          map[string]string
```

Comment:
nit: this doesn't need to be exported it seems like
```suggestion
	ssmParamSecrets          map[string]string
```